source/incqueryserver-jupyter/iqs_jupyter/analysis_extensions.py

```diff
@@ -19,13 +19,14 @@
 
 import html
 
-import iqs_client
```

Comment:
Why was this change necessary? Was there a problem with polluting the namespace?
Issue description #68 tells me nothing.
I do not understand WHY we are changing these.

Reply:
Yes it seems, that the generated client has been changed. And `iqs_client` no longer has the necessary DTO-s. (The decorator may not run, that is why.) My initial goal is to be consistent with the release, however we may need to change the python client as well.